### DISCUSSION TITLE
multi output regression

### DISCUSSION THREAD

**Discussion by paolocalderaro** [timeseriesAI/tsai#474] [Q&A]
good morning, I am looking forward to use tsai to perform a multi-output regression on multi-variate time series. Each of my time series is composed of 10 channel and the target is a vector of 11 real, continuous values. Does tsai accept this use-case or not yet? I found good tutorial about managing MTS and single-output regression problem but not on Multi Target Regression problem.

**Comment by oguiza** [comment]
Hi @paolocalderaro,
You should be able to use tsai for that task out of the box.
Some considerations are:
1. You need to create the target values (y) with the desired shape. tsai learned that shape and auto-configures itself to generate an output with the same shape. 
2. The only models that can auto-configure the output are those ending in Plus. 
I hope tsai works well for you.
